### PR TITLE
Announcement tab overhaul

### LINKED ISSUE #1725: Make announcements into it's own, new tab.
Move the announcement to it's own tab, remove from insert.

### PR DESCRIPTION
**Pull request by theamanjs**
- Fix #1725 
- Removes Insert Tab

### CODE REVIEW

**Comment by Gauravjeetsingh** [comment]
@theamanjs veerji, can you post the screenshot of updated UI here

**Comment by Gauravjeetsingh** [comment]
Few UI issues
image
Guru Hargobind ji is hidden behind footer. 

image
The annoucement tab can use some update, the button can be below the textarea, and textarea can take full width

File: www/src/scss/navigator/misc/_misc.scss [hunk 1]
```diff
@@ -116,55 +116,62 @@ $misc-footer-active: 70px;
...
-        margin: 0 20px 10px;
         min-height: 100px;
         min-width: 300px;
         outline: none;
         padding: 10px;
       }
-      .announcement-slide-btn {
+      &-slide-btn {
         align-self: center;
+        background: #29679b;
```
Comment (Gauravjeetsingh) [inline www/src/scss/navigator/misc/_misc.scss:153]: @theamanjs do we have this color in a variable, if not, should we move it to variable?

File: www/src/scss/navigator/misc/_misc.scss [hunk 2]
```diff
@@ -116,55 +116,62 @@ $misc-footer-active: 70px;
...
         border: 0;
         border-radius: 0;
         font-size: 0.9rem;
         color: #fff;
```
Comment (Gauravjeetsingh) [inline www/src/scss/navigator/misc/_misc.scss:157]: Not a edit in this PR, but came to my attention, we do have a variable for white color which can be used here.

**Review by Gauravjeetsingh — CHANGES_REQUESTED** [review]

**Comment by theamanjs** [comment]
> image The annoucement tab can use some update, the button can be below the textarea, and textarea can take full width

Veerji veerji I've made the change


Screenshot 2023-06-07 at 11 56 54 AM

File: www/main/navigator/misc/components/AnnouncementPane.jsx [hunk 3]
```diff
@@ -57,35 +75,35 @@ export const InsertPane = ({ className }) => {
...
-            <input
-              id="announcement-language"
-              name="announcement-language"
-              type="checkbox"
-              onChange={toggleAnnouncementLanguage}
-            />
-            <label htmlFor="announcement-language" />
+        <div className="announcement-body">
+          <textarea
+            className={`${isMiscSlideGurmukhi ? 'gurmukhi' : ''} announcement-text`}
```
[the remark this answers is not in the document]
Comment (theamanjs) [reply www/main/navigator/misc/components/AnnouncementPane.jsx]: Done veerji

File: www/src/scss/navigator/misc/_misc.scss (hunk 1 above)
Comment (theamanjs) [reply www/src/scss/navigator/misc/_misc.scss]: Done veerji

File: www/src/scss/navigator/misc/_misc.scss (hunk 2 above)
Comment (theamanjs) [reply www/src/scss/navigator/misc/_misc.scss]: Done veerji, Also there was one more hex in this file I have changed that to variable as well. Thanks for pointing out

**Review by Gauravjeetsingh — APPROVED** [review]

File: www/main/navigator/misc/components/MiscHeader.jsx [hunk 4]
```diff
@@ -33,12 +33,12 @@ export const MiscHeader = () => {
       </a>
 
       <a
-        className={`misc-button ${isInsert ? 'misc-active' : ''}`}
-        onClick={() => setTab('Insert')}
+        className={`misc-button ${isAnnouncement ? 'misc-active' : ''}`}
```
Comment (inderpreetsingh) [inline www/main/navigator/misc/components/MiscHeader.jsx:36]: can we use classNames util?

File: www/main/navigator/misc/components/MiscContent.jsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ export const MiscContent = () => {
   return (
     <>
       <HistoryPane className={currentMiscPanel === 'History' ? '' : 'd-none'} />
-      <InsertPane className={currentMiscPanel === 'Insert' ? '' : 'd-none'} />
+      <AnnouncementPane className={currentMiscPanel === 'Announcement' ? '' : 'd-none'} />
```
Comment (inderpreetsingh) [inline www/main/navigator/misc/components/MiscContent.jsx:14]: veerji can you refactor all the places that can use classnames util

**Review by inderpreetsingh — APPROVED** [review]

**Pull request merged**